roles/sap_hostagent/tasks/deploy_bundle.yml

```diff
@@ -1,15 +1,15 @@
 ---
 
 - name: Extract the SAPHOSTAGENT TGZ file from the Bundle
-  unarchive:
+  ansible.builtin.unarchive:
     src: "{{ sap_hostagent_bundle_path }}/HOSTAGENT.TGZ"
     dest: "{{ sap_hostagent_agent_tmp_directory }}"
     remote_src: yes
     mode: '0755'
 
 - name: Install SAPHOSTAGENT
-  command: "{{ sap_hostagent_agent_tmp_directory }}/global/hdb/saphostagent_setup/saphostexec -install"
+  ansible.builtin.command: "{{ sap_hostagent_agent_tmp_directory }}/global/hdb/saphostagent_setup/saphostexec -install"
   register: installagent
   args:
     chdir: "{{ sap_hostagent_agent_tmp_directory }}/global/hdb/saphostagent_setup/"
-  changed_when: "'Install service' in installagent.stdout"
\ No newline at end of file
+  changed_when: "'Install service' in installagent.stdout"
```

Comment:
Is the double quote irrelevant?

Reply:
Probably not. We have the same syntax in all other roles as per output of:
```
$ cd (collection root)
$ grep -rn "changed_when" | grep " in "
```
Also, while working on fixing https://github.com/sap-linuxlab/community.sap_install/issues/555, I got the following error message when removing the double quotes from the `that:` statement in one of the `assert` tasks:
```
  ansible.builtin.assert:
    that: 'bind-utils' in ansible_facts.packages
                       ^ here
This one looks easy to fix. It seems that there is a value started
with a quote, and the YAML parser is expecting to see the line ended
with the same kind of quote. For instance:

    when: "ok" in result.stdout

Could be written as:

   when: '"ok" in result.stdout'

Or equivalently:

   when: "'ok' in result.stdout"
```

Reply:
Thanks for the explanation.